specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/applicationWhitelistings.json

```diff
@@ -265,6 +273,13 @@
         "Enforce"
       ]
     },
+    "ProtectionMode": {
+      "type": "object",
+      "description": "The protection mode of the collection/file types",
+      "additionalProperties": {
+        "type": "string"
+	  }
+    },
```

Comment:
This seems to represent a mapping from file type (enum) to protection mode which seems like an enum also. Generally, using `additionalProperties` is discouraged because it makes APIs harder to use. In you case, you could model each file type as an optional property in this object and let the value be a protection mode enum type. This option would preserve the current shape of the object over the wire.

Another option would be to model this as an array of objects with 2 properties: file type and protection mode.

Reply:
@majastrz  , Done

Reply:
There are few potential new ARM errors, property names should be camel case. See this report https://dev.azure.com/azure-sdk/public/_build/results?buildId=114422&view=logs